internal/pkg/idgenerator/idgenerator.go

```diff
@@ -6,7 +6,7 @@ import gonanoid "github.com/matoous/go-nanoid/v2"
 const (
 	RequestIDLength               = 15
 	TemplateInstanceIDLength      = 25
-	EtcdNamespaceForE2ETestLength = 10
+	EtcdNamespaceForE2ETestLength = 20
```

Comment:
Raz mi spadol test na rovnakom namespace, neviem ci je to mozne, uvidime, ci sa to este stane.

Reply:
Neporadilo sa mi to zopakovat.
V etcd mohol byt nejaky stary namespace, lebo isty cas to mazanie nefungovalo, a ja som conatiners dost dlho nerestartoval, ... a testov mame dost vela.